django_cloud_deploy/workflow/_deploygae.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Workflow for deploying a Django app to GAE."""
+
+import os
+
+import pexpect
+
+
+class DeployNewAppError(Exception):
+    """A class to control the workflow for deploying an Django app to GAE."""
+
+
+class DeploygaeWorkflow(object):
+    """Workflow to deploy Django app on GAE."""
+
+    def deploy_gae_app(self,
+                       project_id: str,
+                       django_directory_path: str,
+                       region: str = 'us-west2') -> str:
```

Comment:
I used 'us-west1' for the default value of region. Is it possible to make default value consistent? I am not sure whether us-west1 support app engine.

Reply:
Talked offline regarding being fine for now